website/common/script/index.js

```diff
@@ -117,6 +121,9 @@ api.shouldDo = shouldDo;
 api.daysSince = daysSince;
 api.DAY_MAPPING = DAY_MAPPING;
 
+api.setDebuffPotionItems = setDebuffPotionItems;
+api.getDebuffPotionItems = getDebuffPotionItems;
```

Comment:
@Xaz16 could these two functions exported under api.libs instead of api directly? thanks!